tinygrad/renderer/cstyle.py

```diff
@@ -273,8 +273,12 @@ class HIPLanguage(CStyleLanguage):
   __attribute__((device)) __attribute__((const)) _Float16 __ocml_sqrt_f16(_Float16);
   }\n""" + '\n'.join([_make_hip_dtype(*x) for x in [("signed int", "int", 2),
                      ("_Float16", "half", 2), ("_Float16", "half", 4), ("_Float16", "half", 8), ("_Float16", "half", 16),
-                     ("float", "float", 2), ("float", "float", 4), ("float", "float", 8)]]) + \
-  'extern "C" __attribute__((global))'
+                     ("float", "float", 2), ("float", "float", 4), ("float", "float", 8)]]) + """
```

Comment:
not specific to this pr but similar to `half_prekernel` i think we want to include these wmma functions only for kernels that have wmma

Reply:
yeah that's a good call.  will setup a PR to fix this in METAL and HIP.